lib/spack/spack/cmd/create.py

```diff
@@ -164,6 +168,10 @@ def install(self, spec, prefix):
         scons('prefix={0}'.format(prefix))
         scons('install')""",
 
+    'bazel': """\
+        # FIXME: Add logic to build and install here.
+        bazel('build')""",
```

Comment:
Are there other widely used packages (other than tensorflow) that build solely with `bazel` ? If not I think it's not worth adding it to `BuildSystemGuesser` & Co. It would be sufficient to code `setup_dependent_module` and `setup_dependent_package` appropriately.